docs/environment.yml

```diff
@@ -32,6 +32,7 @@ dependencies:
   - sphinx >=5.1,<6
   - sphinx-autodoc-typehints
   - sphinx-jsonschema
+  - sphinxcontrib-mermaid
```

Comment:
This might also need to be added to https://github.com/jupyterlite/jupyterlite/blob/main/requirements-docs.txt for the failing CI check to pass.